beta-src/src/webDiplomacyTheme.ts

```diff
@@ -24,9 +51,37 @@ const webDiplomacyTheme = createTheme({
       contrastText: "#000",
     },
   },
-  typography: {
-    fontFamily: "SF Pro Display, Segoe UI, Droid Sans, sans-serif",
-  },
 });
 
+webDiplomacyTheme.typography.body1 = {
+  fontFamily: webDiplomacyTheme.typography.fontFamily,
+  fontSize: 14,
+  fontWeight: 400,
+  lineHeight: 1.2,
+};
+
+webDiplomacyTheme.typography.h1 = {
+  fontFamily: webDiplomacyTheme.typography.fontFamily,
+  fontSize: 16,
+  fontWeight: 700,
```

Comment:
is there a way to make these font-weights and line-heights variables? Maybe we can import a FONTS enum or object and use that within this file?

Reply:
yes, good call. i'll do this before merging.